### DISCUSSION TITLE
Clear cache button disappeared

### DISCUSSION THREAD

**Discussion by 2terNick** [uBlockOrigin/uBlock-issues#3081] [General]
Hello,
Today I noticed that the "Clear cache" button has disappeared.
Opera browser with uBlock Origin 1.55.0

Error or intention?

**Comment by garry-ut99** [comment]
Intention :
- https://github.com/uBlockOrigin/uAssets/issues/22028

**Comment by gorhill** [comment]
Why did you want to use "Purge all caches"? What **specific** issue are you facing that you think "Purge all caches" will solve?

**Reply by 2terNick** [reply]
To block all advertising on YouTube.
At some point the uBlock stops working and then you should delete the uBlock cache so that it can properly block advertising on YouTube again.
This has always worked wonderfully in the past, why is deleting the cache now gone?

**Reply by gorhill** [reply]
Which exact advertising did you get in Youtube? Screenshot? It's possible no filter has been added yet and there is no point to "Purge all caches", which would result in MBs of filter lists to be downloaded from GitHub, just to end up in the same state you were.

The official support thread gives a link to update only the necessary list, and it's more user friendly because a simple click will do everything that is needed to update only the list that is needed:

[Click this link](https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=ublock-quick-fixes) then load Youtube in a new tab. If the issue still exists, [click this link](https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=ublock-quick-fixes&manual=1) then load Youtube in a new tab.

**Reply by gorhill** [reply]
@FuzzyTues follow instructions:

https://old.reddit.com/r/uBlockOrigin/comments/18t9ush/youtube_detection_ads_breakages_2023_12_29_ubo/kib7o8y/

**Comment by gorhill** [comment]
The [volunteer-contributed status page](https://drhyperion451.github.io/does-uBO-bypass-yt/) has been updated to from now on ensure the latest fixes (if present) are guaranteed to be fetched.

This is the official way to update for Youtube -- as long as all the external factors which can also cause detection are being taken care of by following the the other steps at <https://github.com/uBlockOrigin/uAssets/issues/20586>.

**Comment by krystian3w** [comment]
For me works now:

https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=all

If dashboard was closed.

If the link deactivates updates in differential mode, it can be corrected and I will save the corrected one in my bookmarks.
